packages/shared/blockchain/abstract/src/lib/abstract-blockchain.sdk.ts

```diff
@@ -94,6 +95,14 @@ export type FromPrivateKeyOrSignatureId<T extends { fromPrivateKey?: string }> =
   Partial<SignatureId> &
   Partial<Pick<T, 'fromPrivateKey'>>
 
+export type FromPrivateKeyOrSignatureIdOrMnemonic<T extends { fromPrivateKey?: string }> = Omit<
```

Comment:
mnemonic should be inside upper <T extends {fromPrivateKey?:string}> declaration